sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -325,7 +326,7 @@ class kernel_bundle : public detail::kernel_bundle_plain,
 
     std::array<char, sizeof(SCType)> RetValue;
     get_specialization_constant_impl(SpecSymName, RetValue.data());
-    std::memcpy(&Res, RetValue.data(), sizeof(SCType));
+    detail::memcpy(&Res, RetValue.data(), sizeof(SCType));
```

Comment:
This is a host-only API, we can still use `std::memcpy` in here

Reply:
ok